CMakeLists.txt

```diff
@@ -14,6 +14,14 @@ project(httpc)
 
 set(CMAKE_CXX_FLAGS "-std=c++11 -O3")
 
+# Handling of Simple web server with compiler pre-processor macro RHEL_CENTOS_7
```

Comment:
I think better message/comment would be "Set pre-processor macro on RHEL/CentOS 7 for suitable handling in Simple web server code".